src/app/cli/src/client.ml

```diff
@@ -312,11 +345,16 @@ let send_payment =
              ~memo:User_command_memo.dummy
              ~body:(Payment {receiver= receiver_compressed; amount})
          in
-         let txn = User_command.sign sender_kp payload in
-         dispatch_with_message Client_lib.Send_user_commands.rpc
-           [(txn :> User_command.t)]
+         let payment = User_command.sign sender_kp payload in
+         dispatch_with_message Client_lib.Send_user_command.rpc
+           (payment :> User_command.t)
            port
-           ~success:(fun () -> "Successfully enqueued payment in pool")
+           ~success:(fun receipt ->
+             sprintf
+               !"Successfully enqueued payment in pool.\n\
+                 Receipt chain hash of the payment:\n\
+                 %{sexp:Receipt.Chain_hash.t}"
```

Comment:
is this a scalar value or does it have parentheses? We should avoid parenthesized sexp things in CLI output